jobs/WebUi/MasterCI/MasterCI

```diff
@@ -0,0 +1,60 @@
+node{
+    timestamps{
+        withEnv([
+            "branch=${env.branch}",
+            "IS_OFFICIAL_RELEASE=false",
+            "date=current",
+            "timezone=-0500",
+            "BINTRAY_SUBJECT=${env.BINTRAY_SUBJECT}",
+            "PUBLISH=${env.PUBLISH}",
+            "MANIFEST_TYPE=on-web-ui"
+        ]){
+            def message = "Job Name: ${env.JOB_NAME} \n" + "Build Full URL: ${env.BUILD_URL} \n" + "Phase: STARTED \n"
```

Comment:
How we can move the CI logic into on-web-ui ?

can we ?
1.clone on-build-config at the begining of this closure , and save in ```repo_dir```
2. then we can either load("on-build-config/jobs/ShareMethod.groovy")
or call functions like     BuildAndPublish.buildAndPublish() by passing  ```repo_dir``` into it.